pom.xml

```diff
@@ -574,41 +581,74 @@
         <version>${version.com.google.gwt}</version>
       </dependency>
 
-      <!-- ElasticSearch (test scope - for running unit test) -->
+      <!-- Elastic search and transitive dependencies for it. -->
       <dependency>
         <groupId>org.elasticsearch</groupId>
         <artifactId>elasticsearch</artifactId>
         <version>${version.org.elasticsearch}</version>
-        <scope>test</scope>
-      </dependency>
-
-      <!-- ElasticSearch Jest client. -->
-      <dependency>
-        <groupId>io.searchbox</groupId>
-        <artifactId>jest</artifactId>
-        <version>${version.io.searchbox.jest}</version>
       </dependency>
 
       <dependency>
-        <groupId>io.searchbox</groupId>
-        <artifactId>jest-common</artifactId>
-        <version>${version.io.searchbox.jest}</version>
-      </dependency>
-
-      <!-- ElasticSearch requires Lucene version 4.10 instead of 3.X used by IP BOM.
-           NOTE: Only for test scope.-->
-      <dependency>
         <groupId>org.apache.lucene</groupId>
         <artifactId>lucene-core</artifactId>
         <version>${version.org.apache.lucene}</version>
-        <scope>test</scope>
       </dependency>
 
       <dependency>
         <groupId>org.apache.lucene</groupId>
         <artifactId>lucene-queryparser</artifactId>
         <version>${version.org.apache.lucene}</version>
-        <scope>test</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>commons-cli</groupId>
+        <artifactId>commons-cli</artifactId>
+        <version>${version.commons.cli}</version>
+      </dependency>
```

Comment:
There is now no need to copy the whole depMgmt section. It is enough to just override the version property, e.g. in this case you can use `<version.commons-cli>1.3.1</version.commons-cli>`.

This is becuase dashbuilder-parent directly inherits from ip-bom (instead of importing it in depMgmt)